bl/blocks/src/tshape.cpp

```diff
@@ -0,0 +1,10 @@
+#include "tshape.hpp"
+
+std::list<std::pair<int, int>> TShape::getShape() const {
```

Comment:
If you'd put the list as the class member and initlize it in constructor and here just return it here it would be much faster

Reply:
fixed